src/graphml.rs

```diff
@@ -703,9 +732,9 @@ impl GraphML {
 /// :rtype: list[Union[PyGraph, PyDiGraph]]
 /// :raises RuntimeError: when an error is encountered while parsing the GraphML file.
 #[pyfunction]
-#[pyo3(text_signature = "(path, /)")]
-pub fn read_graphml(py: Python, path: &str) -> PyResult<Vec<PyObject>> {
-    let graphml = GraphML::from_file(path)?;
+#[pyo3(signature=(path, compression=""),text_signature = "(path, compression=\"\", /)")]
+pub fn read_graphml(py: Python, path: &str, compression: &str) -> PyResult<Vec<PyObject>> {
```

Comment:
Prefer `Option<&str>` over an empty string. For Python the default would become `None` which gets converted to `None` in Rust as well

Reply:
Yeah I thought about that too, but apparently it's being phased out: 
https://pyo3.rs/main/function/signature#trailing-optional-arguments

Reply:
> Yeah I thought about that too, but apparently it's being phased out: https://pyo3.rs/main/function/signature#trailing-optional-arguments
You just need to specify the None in the signature like you specified the empty string. We still use that in lots of places. What is deprecated is None being inserted by default without declaration